packages/maskbook/src/extension/options-page/DashboardComponents/ProviderLine.tsx

```diff
@@ -56,7 +57,7 @@ export default function ProviderLine(props: ProviderLineProps) {
     const { t } = useI18N()
     const { internalName, network, connected, userId, onAction } = props
     const classes = useStylesExtends(useStyles(), props)
-
+    const isMobile = useMatchXS()
```

Comment:
IMO you should use compile flags. Reason: It's not always true that "small" devices _will_ have a native Twitter/Facebook button to navigate.

Reply:
done